roles/rabbitmq/tasks/main.yml

```diff
@@ -15,3 +15,14 @@
     enabled: yes
     state: started
   tags: rabbitmq
+
+- name: Add RabbitMQ plugins
+  become: yes
+  rabbitmq_plugin:
+    names: "{{ rabbitmq_plugins|join(',') }}"
+    state: enabled
+    new_only: "{{ rabbitmq_plugins_new_only }}"
```

Comment:
After thinking a while, personally I'd prefer to hardcode this to `no` and remove var `rabbitmq_plugins_new_only` for simplicity reasons.

This way Ansible will manage full state of rabbitmq plugins.

WDYT? For your infra, do you need this option as `yes`? I'm OK to leave that as is then.

IMO if we're in infra-as-a-code field, the point of truth is configuration management tool and if any custom plugin is installed manually or by any other system - it should be removed by Ansible to keep in sync with the git/code.

Reply:
For my infra where everything is running on one machine, I don't care. But, will people want this for a HA setup? I can imagine having multiple playbooks logging in to a rabbitmq cluster to set up the users, vhosts, and plugins that are expected for a single app. But you wouldn't want one app removing the info for another app. So, I would err on the side of only removing if specifically requested for multi-tenant rabbitmq HA installs.

Reply:
At the moment we don't focus on any HA-deployment cases for `rabbitmq` role.
See https://github.com/StackStorm/ansible-st2/issues/119 similar discussion.

The current position about `ansible-st2` is that we provide minimal installer similar by logic to `curl|bash` demo installer.
Having said that, at the same time we leave an option to use external roles from galaxy to configure mongo or rabbitmq or postgresql how users want it.

Reply:
The newest commit hard codes `new_only: no`.